solidago/src/solidago/comparisons_to_scores/continuous_bradley_terry.py

```diff
@@ -124,14 +196,25 @@ def compute_individual_scores(self, scores: pd.DataFrame, initial_entity_scores=
             initial_scores = initial_scores.to_numpy()
         theta_star_numpy = self.coordinate_descent(coord_to_subset, initial_scores=initial_scores)
         delta_star_numpy = np.zeros(len(theta_star_numpy))
+        raw_score_lower_bound = np.zeros(len(theta_star_numpy))
+        raw_score_upper_bound = np.zeros(len(theta_star_numpy))
         for idx_a in range(len(theta_star_numpy)):
-            indices_b, _r_ab = coord_to_subset[idx_a]
+            indices_b, r_ab = coord_to_subset[idx_a]
+            lower_bound, upper_bound = get_high_likelihood_range(
+                continuous_bradley_terry_log_likelihood,
```

Comment:
@lenhoanglnh When calculating the high likelihood range, should we include only the likelihood of observed comparisons, or also the prior/regularization (something like `alpha * thetha^2`)?